src/components/Cart/Cart.vue

```diff
@@ -63,7 +67,7 @@ export default {
       toggleMenu: () => store.commit("sideMenu/toggleMenu")
```

Comment:
you could add type of value returned from function, in this case it will be void